src/common/mod.rs

```diff
@@ -24,7 +25,18 @@ enum Focus {
 
 impl<'a, IO: AsyncRead + AsyncWrite, S: Session> Stream<'a, IO, S> {
     pub fn new(io: &'a mut IO, session: &'a mut S) -> Self {
-        Stream { io, session }
+        Stream {
```

Comment:
Maybe we only need `eof`, don't need the whole `TlsState`.

Reply:
Sure. Just saw this, was having a vacation :P